pkg/controller/multi/enforcer.go

```diff
@@ -106,3 +112,25 @@ func (e enforcer) DigestByPolicy(namespace string, img *image.Reference, credent
 
 	return digest, nil, nil
 }
+
+func (e *enforcer) VulnerabilityPolicy(img *image.Reference, credentials credential.Credentials, policy *securityenforcementv1beta1.Policy) vulnerability.ScanResponse {
+	if policy == nil {
+		glog.Infof("No policy for image %q so allow", img.String())
```

Comment:
This should probably be a debug - also could we tag logs with something that indicates where in the policy they came from - so in this case I think it'd be `vulnerability`

Reply:
Done - force pushed to keep to one commit